res/css/views/rooms/_MessageComposer.scss

```diff
@@ -203,6 +203,7 @@ limitations under the License.
 
 .mx_MessageComposer_hangup {
     mask-image: url('$(res)/img/hangup.svg');
+    background-color: #ff4c4c;
```

Comment:
Same line with https://github.com/telred-llc/matrix-react-sdk/pull/11/files
Should separate code from each PR